File: euklid_300v/euklid_300v.py
import time
starta = 2
startb = 1
starttime = time.time()
while (starta <= 1000):
    while (startb <= starta):
        a = starta
        b = startb
        c = 1
        while (a > 0 and b > 0):
            if a < b:
                b = b - a
            else:
                a = a - b

        if a == 0:
            c = b
        else:
            c = a

        print(("ggT von a = " + str(starta) + " und b = " + str(startb) +
        " ist c =  " + str(c)))
        startb = startb + 1
    startb = 1
    print(("---------------------" + str(starta) + "-------------------------"))
    starta = starta + 1
endtime = time.time()

runtime = endtime - starttime
print((str(runtime) + " sec"))

File: euklid_300v/euklid_300v.lua
starta = 2
startb = 1
starttime = os.time()
while (starta <= 1000) do
    while (startb <= starta)do
        a = starta
        b = startb
        c = 1
        while (a > 0 and b > 0) do
            if a < b then
                b = b - a
            else
                a = a - b
              end
            end
        if a == 0 then
            c = b
        else
            c = a
          end
        print("ggT von a = " .. starta .. " und b = " .. startb ..
        " ist c =  " .. c)
        startb = startb + 1
      end
    startb = 1
    print("---------------------" .. starta .. "-------------------------")
    starta = starta + 1
  end
endtime = os.time()

runtime = endtime - starttime
print(runtime .. " sec")
